spec/numberformat.html

```diff
@@ -457,10 +457,11 @@
       <emu-alg>
         1. Let _nf_ be *this* value.
         1. If Type(_nf_) is not Object, throw a *TypeError* exception.
-        1. If _nf_ does not have a [[boundFormat]] internal slot, throw a *TypeError* exception.
-        1. If _nf_.[[boundFormat]] is *undefined*, then
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
+        1. If _nf_.[[initializedNumberFormat]] is *undefined*, then
```

Comment:
I noticed that you didn't change this line for collator and datetimeformat, so I guess this one is just a mistake. Other than this, everything else looks great. I can merge it once you fix this small detail.